### PR TITLE
Update User Guide - Unadvertised resources

### PR DESCRIPTION
**Pull request by ElenaGallo**
## Description

Add Unadvertised option on [Resources properties](https://mapstore--9976.org.readthedocs.build/en/9976/user-guide/resources-properties/) section

### CODE REVIEW

File: docs/user-guide/resources-properties.md [hunk 1]
```diff
@@ -8,23 +8,29 @@ In order to customize the properties of a resource, the Admin or a normal user w
...
 
-* Add a *Permission* rule
+* Add a **Name** and a **Description**
 
-!!! warning
+!!! note
     The name of a resource is the only mandatory field. Note that is not allowed to choose a name that has already been assigned to another resource.
 
-## Thumbnail
+* Enable the **Unadvertised** option to makes the resource owner or the admin the only one who can see and search the resource on the [Home Page](home-page.md) or in the [Map Catalog](map-catalog.md) tool.
```
Comment (tdipisa) [inline docs/user-guide/resources-properties.md:21]:
```suggestion
* Enable the **Unadvertised** option to makes the owner of the resource or the MapStore admin the only ones who can see and search that resource in the [Home Page](home-page.md) or in the [Map Catalog](map-catalog.md) tool.
```

File: docs/user-guide/resources-properties.md [hunk 2]
```diff
@@ -8,23 +8,29 @@ In order to customize the properties of a resource, the Admin or a normal user w
...
 
-It is possible to add an image as thumbnail dropping it or clicking inside the following box:
+* Add a **Permission** rule
```
Comment (tdipisa) [inline docs/user-guide/resources-properties.md:23]: Please put a link to the related section.

File: docs/user-guide/resources-properties.md [hunk 3]
```diff
@@ -8,23 +8,29 @@ In order to customize the properties of a resource, the Admin or a normal user w
...
-<img src="../img/resource-properties/thumb.jpg" class="ms-docimage"/>
+* Add **Details** (only for *Maps* and *Dashboard*)
 
-!!! warning
-    The image to be added must not be larger than 500 kb and its best dimensions are 300x180 px. The supported formats are `jpg` (or `jpeg`) and `png`.
+Once a resource has been saved, in the *Save* panel, the user can view the creation and last modification data of the resource itself. An example in the image below:
+
+<img src="../img/resource-properties/resource_data.jpg" class="ms-docimage"  style="max-width:400px;"/>
+
+While an admin user can also view who created the resource and the last user who modified it. An example in the image below:
```
Comment (tdipisa) [inline docs/user-guide/resources-properties.md:31]:
```suggestion
Admin users can also see who created and modified the resource. An example in the image below:
```

File: docs/user-guide/resources-properties.md [hunk 4]
```diff
@@ -45,11 +51,11 @@ How to manage users and groups is a topic present in the [Managing Users](managi
 
 ## Details
 
-Only for resources of type map, it is possible to add details to the map. This is useful to associate some information to the map or an overview description of its content. In this case the *Edit properties* window is the following:
+Only for resources of type map and dashbaord, it is possible to add details to the resource. This is useful to associate some information to the resource or an overview description of its content. In this case the *Edit properties* window is the following:
```
Comment (tdipisa) [inline docs/user-guide/resources-properties.md:54]:
```suggestion
Only for resources of type _map_ and _dashbaord_, it is possible to add details as additional information in form of descriptive content for the resource itself. This is useful to associate some information to the resource or an overview description of its content. In this case the *Edit properties* window is the following:
```

File: docs/user-guide/resources-properties.md [hunk 5]
```diff
@@ -45,11 +51,11 @@ How to manage users and groups is a topic present in the [Managing Users](managi
...
 
 <img src="../img/resource-properties/edit-map-properties-panel_details.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
-With a click on the **Add new details** button <img src="../img/button/add_details_button.jpg" class="ms-docbutton"/> it opens a panel where the user can write the details of the map.
+With a click on the **Add new details** button <img src="../img/button/add_details_button.jpg" class="ms-docbutton"/> it opens a panel where the user can write the details of the resource.
```
Comment (tdipisa) [inline docs/user-guide/resources-properties.md:58]:
```suggestion
With a click on the **Add new details** button <img src="../img/button/add_details_button.jpg" class="ms-docbutton"/> a panel opens and the user can write the details of the resource.
```

File: docs/user-guide/resources-properties.md [hunk 6]
```diff
@@ -74,7 +80,7 @@ Here, the user is allowed to:
 !!! warning
     The *About this map* button is visible in the [Side Toolbar](mapstore-toolbars.md#side-toolbar) only when the details are present on the map.
 
-* Enable the **Show at startup** <img src="../img/button/show-at-starup-button.jpg" class="ms-docbutton"/> button. If active, as soon as the user opens the map, the details panel is visualized.
+* Enable the **Show at startup** <img src="../img/button/show-at-starup-button.jpg" class="ms-docbutton"/> button. If active, as soon as the user opens the resources, the details panel is visualized.
```
Comment (tdipisa) [inline docs/user-guide/resources-properties.md:83]:
```suggestion
* Enable the **Show at startup** <img src="../img/button/show-at-starup-button.jpg" class="ms-docbutton"/> button. If active, when the user opens a resources with details defined, these are visualized in a descriptive panel.
```

**Review by tdipisa — CHANGES_REQUESTED** [review]
That's good. Only a few corrections in form of suggestions that you can directly apply from github UI.

File: docs/user-guide/resources-properties.md [hunk 7]
```diff
@@ -8,23 +8,29 @@ In order to customize the properties of a resource, the Admin or a normal user w
...
 
-* Add a *Permission* rule
+* Add a **Name** and a **Description**
 
-!!! warning
+!!! note
     The name of a resource is the only mandatory field. Note that is not allowed to choose a name that has already been assigned to another resource.
 
-## Thumbnail
+* * Enable the **Unadvertised** option to makes the owner of the resource or the MapStore admin the only ones who can see and search that resource in the [Home Page](home-page.md) or in the [Map Catalog](map-catalog.md) tool.
```
Comment (tdipisa) [inline docs/user-guide/resources-properties.md:21]:
```suggestion
* * Enable the **Unadvertised** option to make the owner of the resource or the MapStore admin the only ones who can see and search that resource in the [Home Page](home-page.md) or in the [Map Catalog](map-catalog.md) tool.
```

File: docs/user-guide/resources-properties.md [hunk 8]
```diff
@@ -8,23 +8,29 @@ In order to customize the properties of a resource, the Admin or a normal user w
...
 
-It is possible to add an image as thumbnail dropping it or clicking inside the following box:
+* Add a [**Permission** rule](resources-properties.md#permission-rules)
 
-<img src="../img/resource-properties/thumb.jpg" class="ms-docimage"/>
+* Add **Details** (only for *Maps* and *Dashboard*)
 
-!!! warning
-    The image to be added must not be larger than 500 kb and its best dimensions are 300x180 px. The supported formats are `jpg` (or `jpeg`) and `png`.
+Once a resource is saved, the *Edit properties* panel shows the creation and last modification date. An example in the image below:
```
Comment (tdipisa) [inline docs/user-guide/resources-properties.md:27]:
```suggestion
Once a resource is saved, the *Edit properties* panel shows the creation and the last modification dates. An example in the image below:
```

File: docs/user-guide/resources-properties.md [hunk 9]
```diff
@@ -45,11 +51,11 @@ How to manage users and groups is a topic present in the [Managing Users](managi
 
 ## Details
 
-Only for resources of type map, it is possible to add details to the map. This is useful to associate some information to the map or an overview description of its content. In this case the *Edit properties* window is the following:
+Only for resources of type *map* and *dashboard*, it is possible to add details as additional information in form of descriptive content for the resource itself. This is useful to associate some information to the resource or an overview description of its content. In this case the *Edit properties* window is the following:
```
Comment (tdipisa) [inline docs/user-guide/resources-properties.md:54]:
```suggestion
Only for resources of type *map* and *dashboard*, it is possible to add details as additional information in form of descriptive content for the resource itself. This is useful to associate some information to the resource and provide an overview of its content. In this case the *Edit properties* window is the following:
```

**Review by tdipisa — CHANGES_REQUESTED** [review]

**Comment by tdipisa** [comment]
There is also a [failing check on ESLINT](https://github.com/geosolutions-it/MapStore2/actions/runs/8247531208/job/22556153689?pr=9976)

image

Please check and fix. Thank you @ElenaGallo

**Comment by offtherailz** [comment]
@ElenaGallo could you please re-add the rule and fix the indentation of the line.

`docs/user-guide/resources-properties.md` is only the matter of an indentation.

File: .markdownlint.json [hunk 10]
```diff
@@ -3,6 +3,8 @@
   "code-block-style": false,
   "line-length": false,
   "no-inline-html": false,
+  "no-alt-text": false,
+  "ul-indent":false,
```
Comment (offtherailz) [inline .markdownlint.json:7]:
```suggestion
```

File: docs/user-guide/resources-properties.md [hunk 11]
```diff
@@ -8,23 +8,29 @@ In order to customize the properties of a resource, the Admin or a normal user w
...
 
-* Add a *Permission* rule
+* Add a **Name** and a **Description**
 
-!!! warning
+!!! note
     The name of a resource is the only mandatory field. Note that is not allowed to choose a name that has already been assigned to another resource.
 
-## Thumbnail
+* * Enable the **Unadvertised** option to make the owner of the resource or the MapStore admin the only ones who can see and search that resource in the [Home Page](home-page.md) or in the [Map Catalog](map-catalog.md) tool.
```
Comment (offtherailz) [inline docs/user-guide/resources-properties.md:21]:
```suggestion
* Enable the **Unadvertised** option to make the owner of the resource or the MapStore admin the only ones who can see and search that resource in the [Home Page](home-page.md) or in the [Map Catalog](map-catalog.md) tool.
```

**Review by tdipisa — APPROVED** [review]

**Comment by tdipisa** [comment]
@ElenaGallo please backport to 2024.01.xx. Thank you.

**Pull request merged**